Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -500,7 +500,7 @@ protected function replaceVariablesInPhpString(string $phpString): string
             |                                  # or
             (?:(?P<prefix>[a-z]+)\:)           # a prefix followed by : (like "env:")
             )?
-            (?P<name>[A-Z_0-9]+))              # the actual variable name in all upper
+            (?P<name>[A-Za-z_0-9\/]+))         # the actual variable name in all upper
```

Comment:
we're loosening the restrictions for all replacement types here, but the comment still says "variable name in all upper"